quickwit/Makefile

```diff
@@ -16,10 +16,11 @@ fix:
 # `make test-all` starts the Docker services and runs all the tests.
 # `make -k test-all docker-compose-down`, tears down the Docker services after running all the tests.
 test-all:
-	QW_S3_ENDPOINT=http://localhost:4566 \
-	AWS_REGION=us-east-1 \
 	AWS_ACCESS_KEY_ID=ignored \
 	AWS_SECRET_ACCESS_KEY=ignored \
+	AWS_REGION=us-east-1 \
+	QW_S3_ENDPOINT=http://localhost:4566 \
+	QW_S3_FORCE_PATH_STYLE_ACCESS=1 \
```

Comment:
so we never use virtual host style in tests?

Reply:
Not against LocalStack. It did not seem to work.